Fix gradient handling in Canvas class

Ananyasingh2002 opened the pull request:
I added a `getGradient` method to the Canvas class to correctly handle gradient fills in Inkscape extensions. This method retrieves gradient information based on the gradient_id, allowing for proper rendering of gradients in the HTML5 Canvas output.

Ananyasingh2002 committed:
Fix gradient handling in Canvas class

hacktoon commented on ink2canvas/.cph/.canvas.py_d6784c02dbe088b5b55543ee6f44b6d3.prob:
```diff
@@ -0,0 +1 @@
+{"name":"Local: canvas","url":"c:\\Users\\ananya\\OneDrive\\Desktop\\ink2canvas\\ink2canvas\\canvas.py","tests":[{"id":1693586516627,"input":"","output":""}],"interactive":false,"memoryLimit":1024,"timeLimit":3000,"srcPath":"c:\\Users\\ananya\\OneDrive\\Desktop\\ink2canvas\\ink2canvas\\canvas.py","group":"local","local":true}
```
Is this file necessary?

Ananyasingh2002 replied:
The Canvas class and the associated methods you see here are crucial for our project. This class is responsible for handling gradient rendering in our Inkscape extension. Without this code, our extension wouldn't be able to correctly process and display gradient fills from Inkscape SVG files. So, yes, this file is necessary for the proper functioning of our project.

hacktoon replied:
So, can you update the PR to remove this file?

Ananyasingh2002 replied:
Okay

Ananyasingh2002 replied:
I did it

Ananyasingh2002 committed:
Delete ink2canvas/.cph/.canvas.py_d6784c02dbe088b5b55543ee6f44b6d3.prob

I deleted this file(ink2canvas/.cph/.canvas.py_d6784c02dbe088b5b55543ee6f44b6d3.prob).
